### ISSUE TITLE
[A11y][Reflow]links like 'asp.netcore' and the other links are missing when we resize the page to 150% and 175 %

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8186]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1184498
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450)

## Repro Steps:

    Zoom the page to 150%
    Hit the URL and login with appropriate credentials
    Tab till search box, type "JSON" and press enter
    Check whether all the links are clearly visible without any content loss or not

## Actual Results:

Links like 'asp.netcore' and the other links are missing when we resize the page to 150%
   
## Expected Results:
All the links in the page should be clearly visible without any content loss when the page is resized
## User Impact:
User with low vision who need to zoom the page, may face difficultly in understanding, if the content in the page is not clearly visible.

## Notes
The row that contains the download counts, last updated, and tags will cut off with ellipses when the page width is not sufficient to contain all the information. The recommendation here is probably to make it wrap. Note that this occurs even without resizing on packages that contain a lot of tags (see Microsoft.IdentityModel.JsonWebTokens)

**Comment by agr** [comment]
![image](https://user-images.githubusercontent.com/102933/93525172-76f49e80-f8ea-11ea-8c4e-d60968d6da3b.png)
